.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   directories:
     - travis_phantomjs
 before_install:
+  - npm config set registry http://ci.strongloop.com:4873/
```

Comment:
No I meant a different PR for just setting up the travis config:
```
 before_install:
   - npm config set registry http://ci.strongloop.com:4873/
```

If you are not squashing then this also works.